### DISCUSSION TITLE
auto right click[Idea]

### DISCUSSION THREAD

**Discussion by jaybogaybo** [ORelio/Minecraft-Console-Client#1746] [Ideas]
### Prerequisites

- [X] I have read and understood the [user manual](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config)
- [X] I made sure I am running the latest [development build](https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts)
- [X] I tried to [look for similar feature requests](https://github.com/ORelio/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

latest

### Describe your problem

im looking to find out if there is a way to get the players on the client once logged in. to just constantly or delayed if possible. right click whatever is in their hand. i play a server where u have to right click a mask to level up and would be really beneficial if i could use my console client to afk them.
any features i can use to do this or one to be added would be amazing
many thanks

### Suggest a possible solution

a command to just make it start / stop right clicking

### Minecraft Version

1.8.9

### Device

Desktop

### Operating System

Windows

**Comment by ReinforceZwei** [comment]
There is no "auto right click" at protocol level. Different actions will be performed depends on what item is holding. You may want to take a look at the [User Manual #interacting-with-the-minecraft-world](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config#interacting-with-the-minecraft-world).

**Reply by jaybogaybo** [reply]
appreciate the help. where can i find which action and how to script it. I'm new to all this and don't know what to actually use as command to place the head. as presumed that would be the way to go about it

**Reply by jaybogaybo** [reply]
i have found my issue after losts of attempts, it appears it says inventories are not handled for the Mc version. is there anything that i can do to solve this?

**Reply by ReinforceZwei** [reply]
Inventory handling is not supported on 1.8.9.
